src/main/ui/platformNavigation/PlatformNavigationBar.tsx

```
@@ -103,14 +103,12 @@ const PlatformNavigationBar = ({ breadcrumbs }: PlatformNavigationBarProps) => {
               surface={!isMobile}
               footer={
-                isMobile && (
-                  <>
-                    <UserMenuDivider />
-                    <Box component={MenuItem} paddingY={gutters(0.5)}>
-                      <PoweredBy preview />
-                    </Box>
-                  </>
-                )
+                isMobile && [
+                  <UserMenuDivider />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)